brewblox_devcon_spark/device.py

```diff
@@ -79,36 +80,42 @@ def active_profile(self, profile_id: int):
     async def shutdown(self, *_):
         pass
 
-    async def _data_processed(self, processor_func: Coroutine, content: dict) -> dict:
-        # Looks for codec data, and converts it
-        # A type ID is always present in the same dict as the data
-        for path, data in dpath.util.search(content, f'**/{OBJECT_DATA_KEY}', yielded=True):
-
-            # find path to dict containing both type and data
-            parent = '/'.join(path.split('/')[:-1])
-
-            # get the type from the dict that contained data
-            obj_type = dpath.util.get(content, f'{parent}/{OBJECT_TYPE_KEY}')
-
-            # convert data, and replace in dict
-            dpath.util.set(content, f'{parent}/{OBJECT_DATA_KEY}', await processor_func(obj_type, data))
+    async def _process_data(self,
+                            processor_func: codec.TranscodeFunc_,
+                            content: dict
+                            ) -> Awaitable[dict]:
+        containers = [content]
```

Comment:
rename to objects_to_process? containers is a bit confusing

Reply:
fixed